README.md

```diff
@@ -37,6 +38,18 @@ Simply add the following line to your `Cartfile`:
 github "yhirano/LicensePlistViewController"
 ```
 
+### Swift Package Manager
+
+Simply add the following line to your `Package.swift`:
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/yhirano/LicensePlistViewController.git", from: "2.1.3")
```

Comment:
A new release will be needed for proper support (here I assumed the next version will be `2.1.3`). For now, the support can be tested by directly specifying the branch on my fork.
```swift
.package(url: "https://github.com/auramagi/LicensePlistViewController", .branch("feature/spm"))
```